lib/xcake/informative.rb

```diff
@@ -0,0 +1,15 @@
+require 'claide'
+require 'tty'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.